rlw.sh

```diff
@@ -130,6 +130,7 @@ roblox-install () {
 
 playerwrapper () {
 	printf '%b\n' " > begin playerwrapper ()\n---"
+	spawndialog warning "Make sure you've logged in to either Studio or Firefox.\nWithout being logged in, this will make you join as a guest."
```

Comment:
In order to determine if the user is logged in via Firefox or the studio, we would need to do some highly advanced cookie detection which extends far beyond the scope of this project.